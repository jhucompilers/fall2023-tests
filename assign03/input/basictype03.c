int main(void) {
  int i1;
  signed int i2;
  unsigned int i3;

  i1 = -2000000000;
  i2 = -2000000001;
  i3 = 4000000000;

  return 0;
}
