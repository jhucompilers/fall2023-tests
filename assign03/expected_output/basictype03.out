0|main|function|function () returning int
2|i1|variable|int
2|i2|variable|int
2|i3|variable|unsigned int
